packages/react-tabs/package.json

```diff
@@ -10,8 +10,8 @@
     "access": "public"
   },
   "dependencies": {
-    "@hiendv/tabs": "^0.2.7",
-    "octicons-react": "^0.18.9"
+    "@hiendv/tabs": "*",
```

Comment:
* allows lerna to use any version of the package, so it'll always use the local one instead of downloading a version from npm if the version required in `react-tabs` doesn't match the version of `tabs`

Reply:
@villa7 If there are any mismatches with versions, lerna will let us know and we would take actions. Instead of ignoring the problem, I think.